public/api/pins.json

```diff
@@ -0,0 +1,22 @@
+{
+  "pins": [
+    {
+      "id": 1,
```

Comment:
Maybe you want to also add an `imageUrl` field to your locations, to also test how it would work with an image?

Reply:
Need to try that out, true.